src/bin/thoth.rs

```diff
@@ -147,25 +99,62 @@ fn main() -> Result<()> {
             }
         }
         ("account", Some(account_matches)) => match account_matches.subcommand() {
-            ("register", Some(register_matches)) => {
-                let name = register_matches.value_of("name").unwrap();
-                let surname = register_matches.value_of("surname").unwrap();
-                let email = register_matches.value_of("email").unwrap();
-                let password = register_matches.value_of("password").unwrap();
-                let is_superuser = register_matches.is_present("is-superuser");
-                let is_bot = register_matches.is_present("is-bot");
-
+            ("register", Some(_)) => {
                 dotenv().ok();
                 let pool = establish_connection();
-                match register(
-                    &name,
-                    &surname,
-                    &email,
-                    &password,
-                    &is_superuser,
-                    &is_bot,
-                    &pool,
-                ) {
+                let publishers = all_publishers(&pool).expect("Unable to fetch publishers.");
+
+                let name: String = Input::new()
+                    .with_prompt("Enter given name")
+                    .interact_text()?;
+                let surname: String = Input::new()
+                    .with_prompt("Enter family name")
+                    .interact_text()?;
```

Comment:
Should `interact` be used instead of `interact_text`, to support names containing non-ASCII characters? The method description of the latter says it's ASCII-only (although a quick test showed it did seem to be possible to enter non-ASCII characters...)

Reply:
Yes, definitely. I've replaced these with `interact_on` so we also run the prompt in standard output